tests/e2e/transaction/transactionSubmission.test.ts

```diff
@@ -828,4 +829,32 @@ describe("transaction submission", () => {
       expect(submittedTransaction.success).toBe(true);
     });
   });
+
+  describe("transactions with no account authenticator", () => {
```

Comment:
can we add more tests for all the different public key use cases? with a pk, without a pk, with array of pks (some defined some arent), etc